### ISSUE TITLE
[Feature] add basic navigation

### ISSUE THREAD

**Issue by AndrewSav** [bafolts/terraforming-mars#3597]
It would be nice if by clicking on the site header on any page we can be navigated back to the starting page. In particular a link back from `games-overview` would be welcome.

>  In some scenarios you can rely on the browser back button, but in my particular case I have games-overview on my shortcut bar for my server because it's the first page I usually want to see. After that I want to go to the main page. For that I need either add another short cut on the bar which I do not have the space for, or go to the address bar and remove the tail, and this is what I currently have to do. I just wish it was easier, because it's something I do almost every day, sometimes more than once :wink:

May be a uniform header on each page that links to the same places the main page links to could be a solution. @kberg this is following the discord conversation we had.

Thank you for considering this.

**Comment by kberg** [comment]
Yeah, this is a nice idea. I'm sure once inspiration strikes you'll see it
pop up everywhere. I'm knee deep in several projects, but I can see this
getting done.

>
